.prettierrc.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+module.exports = {
+  printWidth: 100,
```

Comment:
😱

Reply:
no? that's what we use at Qonto and what I usually just copy everywhere 😅

Reply:
Nevermind, I'm used to 80 which is maybe because I'm old – I'd be the last to start a discussion around which particular settings we use though so all good 👍